web/templates/setting/form.html.eex

```diff
@@ -0,0 +1,28 @@
+<%= form_for @conn, @action, [as: :settings, method: :put], fn f -> %>
+  <table class="table">
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Value</th>
+      </tr>
+    </thead>
+    <tbody>
+      <%= for svc <- @system_variable_changesets do %>
+        <tr>
+          <td> <%= svc.data.key %> </td>
+
+          <%= inputs_for f, integer_to_atom(svc.data.id), fn ff -> %>
```

Comment:
Seems I fail to see the issue.
whenever you make a form the data is formulated like what I did. in fact I replicated what form generates when it has a valid changeset. what it basically does is creating a map of .   %{id: %{key: key, value: value}}

Anyhow, how to do you suggest me change this ?